spec/security.md

```diff
@@ -1,4 +1,64 @@
 Security Considerations {#sec-security}
 =====================
 
-**TODO**(shaseley): Fill this in.
+<div class="non-normative">
+*This section is non-normative.*
+
+The main security consideration for the APIs defined in this specification is
+whether or not any information is potentially leaked between origins by
+timing-based side-channel attacks.
+</div>
+
+`postTask` as a High-Resolution Timing Source {#sec-security-high-res-timer}
+---------------------
+
+<div class="non-normative">
+The first consideration is whether the {{Scheduler/postTask()}} API can be used
+as a high-resolution timing source. {{Scheduler/postTask()}}'s
+{{SchedulerPostTaskOptions/delay}}, like
+{{WindowOrWorkerGlobalScope/setTimeout()}}'s timeout value, is expressed in
+whole milliseconds (the minimum non-zero delay being 1 ms), and there is no
+guarantee that tasks will run exactly when the delay expires since tasks are
+queued when the delay expires. Given this, we do not believe the API can be
+used as a high-resolution timing source, but we mention it here because of the
+general interest in the topic.
+</div>
+
+Monitoring Another Origin's Tasks {#sec-security-monitoring-tasks}
+---------------------
+
+<div class="non-normative">
+The second consideration is whether {{Scheduler/postTask()}} leaks any
+information about other origins' tasks. The threat model we consider is code
+from two different origins running in separate event loops in the same thread.
```

Comment:
It's not quite right to call this the "threat model". Maybe just "We consider an attacker running on one origin trying to obtain information about code executing in another origin (and hence in a separate event loop) that is scheduled in the same thread in a browser."